modules/virt-creating-bridge-nad-web.adoc

```diff
@@ -1,22 +1,32 @@
 // Module included in the following assemblies:
 //
 // * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
-
 //This file contains UI elements and/or package names that need to be updated.
 
 :_content-type: PROCEDURE
-[id="virt-creating-bridge-nad-web_{context}"]
+[id="virt-creating-linux-bridge-nad-web_{context}"]
 = Creating a Linux bridge network attachment definition in the web console
 
 Network administrators can create network attachment definitions to provide layer-2 networking to pods and virtual machines.
 
 .Procedure
 
 . In the web console, click *Networking* -> *Network Attachment Definitions*.
-. Click *Create Network Attachment Definition* .
+. Click *Create Network Attachment Definition*.
++
+[NOTE]
+====
+The network attachment definition must be in the same namespace as the pod or virtual machine.
+====
++
 . Enter a unique *Name* and optional *Description*.
 . Click the *Network Type* list and select *CNV Linux bridge*.
 . Enter the name of the bridge in the *Bridge Name* field.
 . Optional: If the resource has VLAN IDs configured, enter the ID numbers in the *VLAN Tag Number* field.
 . Optional: Select the *MAC Spoof Check* checkbox to enable MAC spoof filtering. This feature provides security against a MAC spoofing attack by allowing only a single MAC address to exit the pod.
 . Click *Create*.
++
+[NOTE]
+====
+A Linux bridge network attachment definition is the most efficient method for configuring a VLAN interface and provides the most control.
```

Comment:
ditto

Reply:
@phoracek 
Are you saying this should be "Linux bridge node network attachment definition"? I disagree. In the OCP networking docs, a network attachment definition is never called a "node network attachment definition", only network attachment definition. 

Also, the resource is a `NetworkAttachmentDefinition` object. If we call it a "node network attachment definition", that could imply the existence of a `NodeNetworkAttachmentDefinition` object. 
See [Removing an additional network attachment definition](https://docs.openshift.com/container-platform/4.10/networking/multiple_networks/remove-additional-network.html).

Reply:
Sorry, I meant ditto for the other [NOTE], regarding "VLAN interface"

Reply:
I'm confused. Is there another [Note] about VLAN? I don't understand.

Reply:
Second note to rewrite like the one in the previous comment.

Reply:
See next comment, done.